cdap-app-fabric/src/main/java/io/cdap/cdap/internal/events/EventPublishManager.java

```diff
@@ -20,37 +20,49 @@
 import com.google.inject.Inject;
```

Comment:
The spacing for line start seems to be more , please use the Cask scheme for IDE to format.